packages/synapse-react-client/src/lib/containers/download_list_v2/ActionRequiredCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import {
+  Box,
+  Paper,
+  PaperProps,
+  Skeleton,
+  styled,
+  Typography,
+} from '@mui/material'
+import Icon from '../row_renderers/utils/Icon'
+import { StyledComponent } from '@emotion/styled'
+
+export type ActionRequiredCardProps =
+  | {
+      title: React.ReactNode
+      description: React.ReactNode
+      actionNode: React.ReactNode
+      iconType: string
+      count: number
+      isLoading?: false
+    }
+  | { isLoading: true }
+
+const ActionRequiredCardContainer: StyledComponent<PaperProps> = styled(Paper, {
+  label: 'ActionRequiredCardContainer',
+})(({ theme }) => ({
+  padding: `${theme.spacing(2.5)} ${theme.spacing(4)}`,
+  display: 'flex',
```

Comment:
Previous implementation used `grid`, but we can do it just fine with flex with a bit less complexity.